test/fuzz/Liquidator/EndAuctionNoRemainingValue.t.sol

```diff
@@ -73,9 +73,9 @@ contract EndAuctionNoRemainingValue_Liquidator_Fuzz_Test is Liquidator_Fuzz_Test
         uint256 amountLoaned,
         address randomAddress
     ) public {
-        startPriceMultiplier = uint16(bound(startPriceMultiplier, 101, 300));
+        startPriceMultiplier = uint16(bound(startPriceMultiplier, 10_100, 30_000));
         vm.assume(minPriceMultiplier < 91);
-        amountLoaned = bound(amountLoaned, 1, 1000); // No overflow when debt is increased
+        amountLoaned = bound(amountLoaned, 1, 1000); // As we can't set to oracle to return0, we need to be left with dust amount to test.
```

Comment:
we can set registry.minUsdValue, it'll return 0 value for any value lower than the minUsdValue set